### ISSUE TITLE
README.md typo

### ISSUE THREAD

**Issue by Behrooz-Amoozad** [pmwkaa/sophia#143]
Documentation link at the bottom is broken, points to wrong domain.

**Comment by pmwkaa** [comment]
Thanks!
Bottom link to 'Documentation' in https://github.com/pmwkaa/sophia/blob/master/README.md points to
http://sophia.systems/v2.2/index.html, which looks good to me. Is there something else?

**Comment by Behrooz-Amoozad** [comment]
yes. one of the links was 'sphia' without 'o'. got me to a russian domain
retailer.

**Comment by Behrooz-Amoozad** [comment]
Hi.
Well, I looked at it again, I was wrong about which readme file it was.
Just searched the repository for sphia.org and found a few hundred. think they should be replaced since the domain is expiring. at least, now they redirect to sophia.systems so others don't get confused.

Great job, btw. I really enjoy reading your code.
